screens/WelcomePage.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+
+export default function HomePage({ navigation }) {
+  return (
+    <View className="flex-1 items-center justify-center bg-gray-100">
+      <LinearGradient
+        colors={['#D8D9CE', '#FFFFFF']}
+        className="absolute inset-x-0 top-[26%] w-full h-[80%] rounded-t-[40px]"
+      />
+
+      <Image
+        source={require('../assets/kebab_logo.png')}
+        className="w-40 h-40 mb-5 z-10"
+        resizeMode="contain"
+      />
+
+      <Text className="text-2xl font-bold text-center mb-4 z-10 text-gray-800">
+        LEGNICA KEBAB CITY TOUR
```

Comment:
Here you can use the uppercase from tailwind instead of writing it by hand

for example: ``` <p class="uppercase ...">The quick brown fox ...</p> ```

Reply:
Done